managed/cmd/pmm-managed/main.go

```diff
@@ -805,7 +810,17 @@ func main() {
 		l.Fatalf("Could not create Percona Portal client: %s", err)
 	}
 
-	telemetry, err := telemetry.NewService(db, platformClient, version.Version, cfg.Config.Services.Telemetry)
+	uieventsService, err := uievents.New()
+	if err != nil {
+		l.Fatalf("Could not create UI Events Service: %s", err)
+	}
+	uieventsService.ScheduleCleanup(ctx)
+
+	telemetryExtensions := map[telemetry.ExtensionType]telemetry.Extension{
+		telemetry.UIEventsExtension: uieventsService,
+	}
```

Comment:
Why it's made this way? Why do we need it pluggable?

Reply:
yes, it should be configurable

Reply:
But why?

Reply:
It should be configurable so that it is explicit what is being collected. Plus we need to be able to remove collection is needed.